docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -752,11 +1055,28 @@ were misclassified (`actual_class` does not match `predicted_class`):
 the predicted class.
 ====
 
-When you have trained a satisfactory model, you can deploy it to make predictions
-about new data. Those steps are not covered in this example. See
-<<ml-inference>>.
-
 If you don't want to keep the {dfanalytics-job}, you can delete it in {kib} or
 by using the {ref}/delete-dfanalytics.html[delete {dfanalytics-job} API]. When
 you delete {dfanalytics-jobs} in {kib}, you have the option to also remove the 
 destination indices and index patterns.
+
+When you have trained a satisfactory model, you can deploy it to make 
+predictions about new data by using {infer}.
+
+[discrete]
+[[ml-inference-class]]
+== {infer-cap}
```

Comment:
> == {infer-cap}

IMO this section should be moved above the example (perhaps embedded in a "Deploying the model" section?